parquet-common/src/main/java/org/apache/parquet/bytes/ByteBufferInputStream.java

```diff
@@ -49,6 +59,19 @@ public static ByteBufferInputStream wrap(List<ByteBuffer> buffers) {
     }
   }
 
+  public static ByteBufferInputStream wrap(ByteBuffer buffer, int offset, int count) {
```

Comment:
What is the purpose of these methods? We already wrap buffers, why not have the caller handle this?

It's also really strange to pass a buffer with an offset and a "count" (do you mean length?) because buffers already have that information built in.

Reply:
This was to make ByteBufferInputStream resemble ByteBuffer. ByteBuffer has methods like this. I've gone ahead and removed them at your request.